src/test/java/org/pesmypetcare/webservice/dao/communitymanager/ForumDaoTest.java

```diff
@@ -421,17 +421,18 @@ public void reportMessage() throws DatabaseAccessException, DocumentException,
                             eq("creator"), eq(username), eq(publicationDate), eq(date));
                     verify(batch).update(same(documentReference), eq("reportedBy"),
                         eq(FieldValue.arrayUnion(username2)));
-                    verify(batch).update(same(documentReference), eq("banned"),
-                        eq(true));
+                    verify(batch).update(same(documentReference), eq("banned"), eq(true));
                 }
 
                 @Test
-                public void unbanMessage() throws DatabaseAccessException, DocumentException {
+                public void unbanMessage() throws DatabaseAccessException, DocumentException, InvalidOperationException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 121).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5638522)